pkgs/development/libraries/imgui/default.nix

```diff
@@ -41,9 +41,27 @@
   IMGUI_FREETYPE ? false,
   IMGUI_FREETYPE_LUNASVG ? false,
   IMGUI_USE_WCHAR32 ? false,
-}@args:
+  # Enable docking and multi-viewports features[^2]
+  # [^2]: https://github.com/ocornut/imgui/wiki/Docking
+  dockingSupport ? false,
```

Comment:
Thanks for the contribution and for volunteering to maintain the package! Reflecting if we really want a boolean argument here, or just `overrideAttrs (_: { src = ...; })`

Reply:
I think there are enough programs that use the docking version of ImGui that creating an option for it so that not every dependent package has to do `overrideAttrs` and maintain their own up to date version of ImGui would be worthwhile